index.js

```diff
@@ -47,9 +44,31 @@ class ExecutorQueue extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        return this.connectBreaker.runCommand()
+        return this.breaker.runCommand('connect')
             // Note: arguments to enqueue are [queue name, job type, array of args]
-            .then(() => this.enqueueBreaker.runCommand('builds', 'start', [config]));
+            .then(() => this.breaker.runCommand('enqueue', this.buildQueue, 'start', [config]));
+    }
+
+    /**
+     * Stop a running or finished build
+     * @method _stop
+     * @param {Object} config               Configuration
+     * @param {Object} [config.annotations] Optional key/value object
+     * @param {String} config.buildId       Unique ID for a build
+     * @return {Promise}
+     */
+    _stop(config) {
+        return this.breaker.runCommand('connect')
+            .then(() => this.breaker.runCommand('del', this.buildQueue, 'start', [config]))
+            .then((numDeleted) => {
+                if (numDeleted !== 0) {
```

Comment:
What if `numDeleted > 1` ? If there are multiple start builds in queue will we end up cancelling all ? Should a single stop event affect a single start event ?